File: FrontendRazorPage/Models/AdminModel/LessonAdminModel.cs
namespace FrontendRazorPage.Models.AdminModel
{
    public class LessonAdminModel
    {
        public int Id { get; set; }
        public int LevelId { get; set; }
        public string Title { get; set; } = string.Empty;
        public int Order { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}

File: FrontendAngular/src/app/Models/AdminModel/lesson-admin-model.ts
export interface LessonAdminModel {
    id: number;
    levelId: number;
    title: string;
    order: number;

    // Thường Admin sẽ cần thêm 2 trường thời gian này để quản lý
    createdAt: string;
    updatedAt: string;
}